java/google/registry/rde/imports/RdeImportUtils.java

```diff
@@ -83,9 +101,15 @@ public RdeImportUtils(
         type,
         resource.getForeignKey(),
         resource.getRepoId());
-    ofy.save().entity(resource);
-    ofy.save().entity(ForeignKeyIndex.create(resource, resource.getDeletionTime()));
-    ofy.save().entity(EppResourceIndex.create(Key.create(resource)));
+    return ImmutableSet.<Object>of(ForeignKeyIndex.create(resource, resource.getDeletionTime()),
+        EppResourceIndex.create(Key.create(resource)));
+  }
+
+  private <T extends EppResource> void importEppResource(final T resource, final String type) {
```

Comment:
Same comments regarding generic type parameter and also the "type" method parameter.